### PR TITLE
Power shell commandlets for the Azure RBAC

### PR DESCRIPTION
**Pull request by rakashya**
Power shell cmdlets to support the RBAC custom roles in Azure.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@rakashya__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (rakashya). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by ogail** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@rakashya - there are merge conflicts, please update

File: src/ResourceManager/Batch/Commands.Batch.Test/Commands.Batch.Test.csproj [hunk 1]
```diff
@@ -60,16 +60,15 @@
     <Reference Include="Microsoft.Azure.Gallery">
       <HintPath>..\..\..\packages\Microsoft.Azure.Gallery.2.6.2-preview\lib\net40\Microsoft.Azure.Gallery.dll</HintPath>
     </Reference>
-    <Reference Include="Microsoft.Azure.Management.Authorization">
+    <Reference Include="Microsoft.Azure.Management.Authorization, Version=0.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```
Comment (markcowl) [inline src/ResourceManager/Batch/Commands.Batch.Test/Commands.Batch.Test.csproj]: This is a big problem - thgis will cause failures in upgrade scenarios - please look at the assembly and file version for your published nuget library

**Pull request merged**